code/modules/antagonists/gang/gang.dm

```diff
@@ -0,0 +1,199 @@
+/datum/antagonist/gang
+	name = "Family Member"
+	roundend_category = "gangsters"
+	var/gang_name = "Leet Like Jeff K"
+	var/gang_id = "LLJK"
+	var/datum/team/gang/my_gang
+	var/list/acceptable_clothes = list()
+	var/list/free_clothes = list()
+	var/datum/action/cooldown/spawn_induction_package/SIP = new()
```

Comment:
Bad variable name.

Reply:
it's literally just Spawn Induction Package abbreviated

Reply:
And? It's still bad even just because it's all capital letters and not recognizable at a glance.

Reply:
i just realized this is a definition and not a mid proc temp, im an idiot lol

ye ill rename it